orttraining/orttraining/core/optimizer/localized_recompute.cc

```diff
@@ -36,6 +36,12 @@ Status GeluRecompute::ApplyImpl(Graph& graph, bool& modified, int /*graph_level*
       continue;
     }
 
+    std::vector<NodeArg*> gelu_input = {node.MutableInputDefs()[0]};
```

Comment:
Just a for loop to go through all the inputs? 
By default, the recompute node should copy everything from the original node.

Reply:
I think we don't even need a loop or copy, now I just pass `MutableInputDefs()` to `AddNode()`.
Any potential issues I'm missing?

Reply:
yes. MutableInputDefs() is good enough